source/rle.js

```diff
@@ -1,7 +1,33 @@
 'use strict';
+	
+/**
+  * Сжимаем строку по алгоритму rle с использованием регулярных выражений
+  * 
+  * @param {string} string - исходная строка
+  * @returns {string} - сжатая строка
+  */
+const rleRegExp = (string) => (typeof string != 'string') ? ''
+    :string.replace(/(.)\1+/g, (series, char) => (char + series.length));
```

Comment:
давай лучше так перенесем
```suggestion
    condition 
       ? a
       : b
```

Reply:
пробел после `:` потерялся, и, раз все равно править, давай еще отступы добавим перед ? и :